docs/topics_5/web-openshift-troubleshoot.adoc

```diff
@@ -83,15 +83,15 @@ When accessing the executor Pod's logs the following error is seen.
 
 *What it means:*
 
-A `NoRouteToHostException` indicates that this Pod is not able to connect to the {WebName} Pod.
+A `NoRouteToHostException` indicates that this pod is not able to connect to the `{WebName}` pod.
 
 *How to resolve it:*
 
-In a new deployment this is expected, as the executor starts before the {WebName}. Otherwise, examine the {WebName} Pod and resolve any errors seen here. Once the {WebName} is running successfully this error should be resolved.
+In a new deployment this is expected, as the executor starts before the {WebName}. Otherwise, examine the `{WebName}` pod and resolve any errors seen here. Once the {WebName} is running successfully this error should be resolved.
 
-.Pod reports insufficient resources
+.A pod reports insufficient resources
```

Comment:
Which pod? Any pod? I suspect it is the `mta-web-console` pod. Please confirm the pod that requires these resources. Ask Marco, Jonathan, or Phil.